abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
Yes sorry I meant in my *pseudo code* a byte content checking. And what you wrote is clear in my head but I'm just not so a fan of having this behavior of  `(Set)Bytes()`. Using `big.Int` it stays consistent: https://play.golang.org/p/pm3eAfoKYg
But anyway, as i'm clearly alone in my opinion, I already did the changes and that's fine, I'll still be able to sleep at night ;)